vocabularies/Org.OData.Capabilities.V1.xml

```diff
@@ -563,6 +563,9 @@ supported:
           <Annotation Term="Core.Description"
             String="The maximum number of navigation properties that can be traversed when addressing the collection to insert into. A value of -1 indicates there is no restriction." />
         </Property>
+        <Property Name="Permissions" Type="Capabilities.Permission">

```

Comment:
Name is plural, type is single-valued: 
- either "Permission" single-valued, 
- or "Permissions" Type="Collection(...)"

Reply:
Name=Permission, Type=...PermissionType